examples/shapewin.py

```diff
@@ -165,7 +165,7 @@ def loop(self):
                                            self.sub_pm)
 
             # Shape has changed
-            elif e.type == self.d.extension_event.ShapeNotify:
+            elif e.type == shape.Event.Notify:
```

Comment:
Perhaps adding the new autogenerated events scoped under the extension name?  E.g. `self.d.extension_event.Shape.NotifyEventData`

(Though that whole dynamic creation of classes and methods seems like a mistake 16 years later...  It should be rethought to allow extensions to be implemented as separate packages, which would also avoid the whole license issue - if that's still an issue when the extensions can be autogenerated.)